tests/test_retrieve_existing.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from flask import Blueprint, jsonify, request, json
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm.exc import NoResultFound
```

Comment:
I'm assuming these were imported to support the code that's been commented out below. Assuming you delete it, the unused imports should be removed as well.